ios/Classes/CountlyFlutterPlugin.m

```diff
@@ -1408,6 +1414,10 @@ - (void)populateConfig:(NSDictionary *)_config {
         if (loggingEnabled) {
             config.enableDebug = [loggingEnabled boolValue];
         }
+        NSNumber *locationDisabled = _config[@"locationDisabled"];
+        if (locationDisabled) {
+            [Countly.sharedInstance disableLocationInfo];
```

Comment:
Can you please re-verify this "disableLocationInfo" when setting during config.
as internally this call check the consent for location also sent a request for location, most probably it will not work before init.

Reply:
Yeah, there is no method to disable location on CountlyConfig.m file. This is the only disable location call on iOS.

Reply:
If this function is not doing anything during config for iOS then we need to mention about this thing in comment of "setDisableLocation/disableLocation" function/
Something like that:
"This function is only for Android"